.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+on:
+  push:
+  pull_request:
+    types: [opened]
+
+name: CI
+
+jobs:
+  test:
+    name: build test
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [
+          #ubuntu-18.04,
+          macos-10.15,
+          windows-2019
+          ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: ubuntu dependencies
+        if: startsWith(matrix.os,'ubuntu')
+        run: |
+          sudo apt-get update && \
+          sudo apt-get -y install \
+            openssl \
+            libssl-dev \
+            pkg-config \
+            libsqlite3-dev \
+            git \
+            cmake \
+            zip \
+            libc++-dev \
+            libc++abi-dev \
+            libprotobuf-dev \
+            protobuf-compiler
+
+      - name: macOS dependencies
+        if: startsWith(matrix.os,'macos')
+        run: brew install cmake zip
+
+      - name: windows dependencies
+        if: startsWith(matrix.os,'windows')
+        run: |
+          vcpkg.exe install sqlite3:x64-windows zlib:x64-windows
+          choco upgrade openssl llvm zip psutils -y
+
+      - name: windows env
+        if: startsWith(matrix.os,'Windows')
+        uses: allenevans/set-env@57c4e41b5835146bcd9889174001783dd682b6c5
+        with:
+          SQLITE3_LIB_DIR: 'C:\vcpkg\installed\x64-windows\lib'
+          OPENSSL_DIR: 'C:\Program Files\OpenSSL-Win64'
+          LIBCLANG_PATH: 'C:\Program Files\LLVM\bin'
```

Comment:
Should this not be `C:\Program Files\LLVM\lib`? That is how it is on my system.

Reply:
Did you also use `choco` to install? I'm not sure about the difference, adding this got the build working on github actions for me.

Reply:
Interesting...

![image](https://user-images.githubusercontent.com/39146854/100198241-3f1a6080-2f04-11eb-8c7c-951188bf1b50.png)

![image](https://user-images.githubusercontent.com/39146854/100198281-4e99a980-2f04-11eb-8038-6b7af3f72215.png)

Reply:
That is strange, because on my windows machine I used scoop and also had to set LIBCLANG_PATH to the `lib` folder 🤷‍♂️